magefiles/image/production.go

```diff
@@ -77,7 +80,10 @@ func (image *Image) ProductionClean() *Image {
 	image = image.OK()
 
 	if os.Getenv("DEBUG") != "" {
-		image.container.Export(image.ctx, "tmp/app.prod.tar")
+		_, err := image.container.Export(image.ctx, "tmp/app.prod.tar")
+		if err != nil {
+			sysexit.Create(err)
+		}
```

Comment:
Something we could do to simplify the duplication that's done in some Go codebases is:

```suggestion
		must(err)
```

Where there's then:

```go

func must(err error) {
	if err != nil {
		sysexit.Create(err)
	}
}